.circleci/config.yml

```diff
@@ -377,7 +377,8 @@ jobs:
           command: |
             export NODE_OPTIONS=--max_old_space_size=1024
             export CCI_MAXCPU=4
-            ./makerpm.sh -a -d -S tools/packages/opennms/opennms.spec
+            . .circleci/scripts/configure-signing.sh
```

Comment:
Secrets are not set when building forked PRs (in order to restrict access to secrets to committers), so we'll need to make the signing conditional.

Depending on the overhead of signing the packages, we may also want to limit it further.

Reply:
Overhead is very low, just a few extra seconds on the makerpm.  I'll fix them to be conditional though.